frontend/src/components/FacilityBooking/BlockStyles.ts

```diff
@@ -0,0 +1,106 @@
+import styled, { css } from 'styled-components'
+import { TimeBlockType } from '../../store/facilityBooking/types'
+
+export const blockHeight = 90
+export const topDistance = 10
+export const blockGap = 5
```

Comment:
File variables should be UPPERCASE  `BLOCK_HEIGHT`

Reply:
[0f3d064](https://github.com/rhdevs/rhapp/pull/597/commits/0f3d06453ab5985054d96df8a65b6542eea3dd94)